Initial working GUI (#9)
canonical/sdcore-gui: README.md

Reviewer: Is the `Proof of Concept` warning needed?

Author: Removed.

  <h1>SD-Core GUI</h1>
</div>

> Graphical User Interface (GUI) for operating the core (creating subscribers, network slices, device groups).

Reviewer 2: ```suggestion
Graphical User Interface (GUI) for configuring SD-Core and managing subscribers.
```

Author: Changed.

```

Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.

> ### 🪨 Rock Container Image

Reviewer 2: Most of the stuff here should be in a CONTRIBUTING.md, not really in the README

Author: I've removed the ROCK part, this will be added in the ROCK PR. I've moved the rest to CONTRIBUTING.md

1. Install rockcraft

```bash

> sudo snap install rockcraft --channel=latest/edge --classic

Reviewer 2: Probably bundle all the installs at first (rockcraft, skopeo, docker)

Author: I've removed, the ROCK part, it will be added in his PR.

```

2. Build the image

```bash

> rockcraft pack

Reviewer: where is the rockcraft.yaml that is being packed?

Author: I've removed this part. I'll add it in the next PR for the Rock.
